apps/edxapp/templates/cms/job_05_load_fixtures.yml.j2

```diff
@@ -1,3 +1,4 @@
+{% if env_type in trashable_env_types %}
```

Comment:
WDYT of adding a specific flag in configuration because some people might use this example course differently and not load it as all or load it always...

Reply:
:thinking: that's a good idea. I'll declare an issue to address this later.